app/src/main/java/com/google/android/sambadocumentsprovider/nativefacade/SambaFacadeClient.java

```diff
@@ -134,14 +139,37 @@ public void rmdir(String uri) throws IOException {
 
   @Override
   public SmbFile openFile(String uri, String mode) throws IOException {
+    return new SambaFileClient(mHandler.getLooper(), openFileRaw(uri, mode));
+  }
+
+  @Override
+  public ParcelFileDescriptor openProxyFile(
+          String uri,
+          String mode,
+          StorageManager storageManager,
+          ByteBuffer buffer,
+         CancellationSignal signal) throws IOException {
```

Comment:
nit: indentation.

Actually there is a way to avoid putting StorageManager stuff in here using factory method, but it's not worthy it.